.changeset/tiny-starfishes-scream/changes.json

```diff
@@ -0,0 +1,4 @@
+{
+  "releases": [{ "name": "better-ajv-errors", "type": "minor" }],
```

Comment:
Sorry, can you please change it to `patch`. Since we are still on `0.x.x`.

Reply:
Ah, sorry, I meant to release patch obviously. Changing it